__tests__/app.test.js

```diff
@@ -250,3 +250,27 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE: delete a comment by accessing it by id", () => {
+  test("204: removes a comment from the comment table", () => {
+    return request(app).delete("/api/comments/3").expect(204);
+  });
+  test("400: bad request", () => {
```

Comment:
This is an unmatched endpoint and so this test isn't really specific to this PATCH request. I imagine you have an app.all handling wildcard routes? 

This should also be a 404 Not Found rather than a 400